### ISSUE TITLE
Dev - Title Missing on Voting Page

### ISSUE THREAD

**Issue by johnstew** [webpack/webpack.js.org#720]
The title is missing on the [vote page](https://webpack.js.org/vote/). 

~Small~ issue but good for SEO.

**Comment by pllearns** [comment]
Hey @johnstew, I can take this PR. What should the title be? Influence or Vote? I agree that this is good for SEO.

**Comment by simon04** [comment]
Why does this change not set the HTML title in `build/vote/index.html`?
```diff
diff --git a/antwar.config.js b/antwar.config.js
index aa174f68..1cb9c884 100644
--- a/antwar.config.js
+++ b/antwar.config.js
@@ -150,24 +150,28 @@ module.exports = {
     ),

     vote: {
+      title: 'Vote',
       path() {
         return require('./components/vote/list.jsx').default
       }
     },

     'vote/feedback': {
+      title: 'Vote',
       path() {
         return require('./components/vote/list.jsx').default
       }
     },

     'vote/moneyDistribution': {
+      title: 'Vote',
       path() {
         return require('./components/vote/list.jsx').default
       }
     },

     organization: {
+      title: 'Organization',
       path() {
         return require('./components/organization/organization.jsx').default
       }
```

**Comment by pllearns** [comment]
Agreed with the idea @simon04, but it appears from what I see (exposing some ignorance) that the vote is not set-up in the paths/section area of the config. It may be a file structure thing too. But again, exposing ignorance. I'll submit two PRs to address through styling and HTML.

**Comment by simon04** [comment]
https://webpack.js.org/vote/ still has no HTML title set:
```html
<!DOCTYPE html>
<html>
  <head>
    <meta charset="UTF-8">
    <title></title>
<!-- ... -->
```

Help on https://github.com/webpack/webpack.js.org/issues/720#issuecomment-275931720 is appreciated.

**Comment by bebraw** [comment]
@simon04 Look into `antwar.config.js`. It picks up the static title from there. There are sections like `vote: {`. If you add `title: 'foobar'` etc., the build should work. It's the same idea for the other sections too.

**Comment by simon04** [comment]
That's what I tried in https://github.com/webpack/webpack.js.org/issues/720#issuecomment-275931720. Am I missing/overlooking something?

**Comment by bebraw** [comment]
@simon04 The titles are taken into account only for the final build.

**Comment by simon04** [comment]
Final build is `npm run build`? I did that. Afterwards `build/vote/index.html` does contain an empty `<title></title>`.

**Comment by bebraw** [comment]
@simon04 Ok, it might be a bug in the generator then. It probably applies the template a little differently in this particular case (different `path()` definition).

**Comment by pllearns** [comment]
Agreed, my approaches were cosmetic, but, a different path definition should work. Is there a way that the vote page could be added to the section in `antwar.config.js` where the other sections live (i.e. `concepts`, `guides`, `configuration`? Apologies I couldn't help more right now. I'll check in later this week.

**Comment by skipjack** [comment]
Created an issue over at antwarjs/antwar#127 for this with more details.

**Comment by skipjack** [comment]
@bebraw seeing as antwarjs/antwar#127, can you test and close this once you release and bump the antwar version?

**Comment by bebraw** [comment]
@skipjack This should be fixed when #980 goes in.

**Comment by skipjack** [comment]
Closing as I added this to the list of TODOs in #1380 -- will either be addressed by #980 or a port.
